google/cloud/security/scanner/scanners/cloudsql_rules_scanner.py

```diff
@@ -42,10 +44,10 @@ def _get_project_policies(self):
             dict: If successful returns a dictionary of project policies
         """
         project_policies = {}
-        project_policies = (
-            project_dao.ProjectDao().get_project_policies('projects',
-                                                          self.\
-                                                          snapshot_timestamp))
+        project_policies = (project_dao
+                            .ProjectDao(self.global_configs)
```

Comment:
Which is the way that you prefer?  here or in the bucket_scanner?  I can make a quick pass to get them to be consistent.